markets/spot-market/subgraph/optimism-mainnet/handleOrderCommitted.ts

```diff
@@ -16,13 +15,15 @@ export function handleOrderCommitted(event: OrderCommitted): void {
   order.block = event.block.number;
   order.timestamp = event.block.timestamp;
 
-  addClaimToOrder(
-    order,
-    event.address,
-    event.params.marketId,
-    event.params.asyncOrderId,
-    'Commited'
-  );
+  order.status = 'Commited';
+
+  // TODO: order claim may need to be added to OrderCommitted event again (used to be there)
+  // let claim = event.params.asyncOrderClaim;
+  // order.amountEscrowed = claim.amountEscrowed;
+  // order.settlementStrategyId = claim.settlementStrategyId;
+  // order.settlementTime = claim.settlementTime;
+  // order.minimumSettlementAmount = claim.minimumSettlementAmount;
+  // order.settledAt = claim.settledAt;
```

Comment:
Requires someone with more context than me to review this properly. I would think we should define what's  required data and then remove unneeded things from the schema and add required things to the event?